### ISSUE TITLE
PLEASE ADD iPHONE 5s 10.3.1 SUPPORT

### ISSUE THREAD

**Issue by IWANNAJAILBREAK** [iabem97/saigon#140]
IF YOU DO EVERYONE WILL LOVE YOU JAILBREAKING SCENE HAS BEEN DEAD

**Comment by fugiefire** [comment]
The jailbreak only works up to ios 10.2.1....

**Comment by matteodalessio** [comment]
This is not a support forum, instead it's for issues with the app or the jailbreak!

**Comment by fugiefire** [comment]
True, but we can add requests here as well because where else can we? @iabem97 said to use reddit for issues only.

**Comment by Samgisaninja** [comment]
@fugirfire don’t request that he add 10.3.1 support to a 10.3.1 jailbreak. 

I asked him a while back, “can you push to github so that devs can look at it” and he said no because he didn’t want to be flooded by GitHub issues that aren’t actually issues. Clearly he was right.

**Comment by fugiefire** [comment]
@Samgisaninja I am not requesting that he add support for other versions, what i said was that this is pretty much the only place where we can put requests or questions. @IWANNAJAILBREAK is the one asking for added support. What I asked in my request/question was if other exploits, such as 
[async_wake-master.zip](https://github.com/iabem97/saigon/files/1566020/async_wake-master.zip) that, were run before the other exploits and post jailbreak scripts that would have required higher privs, if it could make the jailbreak work with higher versions of iOS.
